debezium-connector-sqlserver/src/main/java/io/debezium/connector/sqlserver/SqlServerOffsetContext.java

```diff
@@ -21,13 +21,15 @@
 
     private static final String SERVER_PARTITION_KEY = "server";
     private static final String SNAPSHOT_COMPLETED_KEY = "snapshot_completed";
+    private static final String OPERATION_ORDER_KEY = "order";
 
     private final Schema sourceInfoSchema;
     private final SourceInfo sourceInfo;
     private final Map<String, String> partition;
     private boolean snapshotCompleted;
+    private int operationOrder;
```

Comment:
Is this change backward-compatible? What steps are needed to migrate existing connectors?

Reply:
@grzegorz8 It is, this is handled via https://github.com/debezium/debezium/pull/915/files#diff-2866cd560785c5b379a9b1ab0d312748R162 When the order is not present in the offsets it will set it to zero forcing the replay of the last stored event.